confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/DeleteStorageGroupsReq.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.confignode.consensus.request.write;
+
+import java.util.List;
+import java.util.Objects;
+
+public class DeleteStorageGroupsReq {
```

Comment:
Should extends ConfigRequest.

Reply:
This request does not work through consensus layer. Only _deleteStorageGroupProcedure_ needs to write consensus.

Reply:
Better to distinguish between requests that send to consensus and those sent to the procedure.

We could rename this to DeleteStorageGroupsProcedureParameter or so

Reply:
Now it is replaced by a _List_.